src/main/scala/io/iohk/praos/crypto/package.scala

```diff
@@ -73,34 +42,8 @@ package object crypto {
     */
   def combineSeeds(x: Seed, y: Seed): Seed = abs(x + y)
 
-  /**
-    * A calculator of hash values.
-    */
-  trait Hasher {
-
-    /** Message digest. */
-    type Digest = ByteString
-
-    /**
-      * Hashes a message.
-      *
-      * @param message  The message to be hashed.
-      * @return  The message digest.
-      */
-    def hash(message: ByteString): Digest
-  }
-
-  /**
-    * A pre-defined hasher.
-    *
-    * @param algorithm  A pre-defined algorithm (e.g. "MD5").
-    * @note  Precondition: The given algorithm is supported.
-    */
-  case class PredefinedHasher(algorithm: String) extends Hasher {
-    override def hash(message: ByteString): Digest = {
-      ByteString(java.security.MessageDigest.getInstance(algorithm).digest(message.toArray))
-    }
-  }
+  // TODO: Change to a realistic value.
+  def seedNonce: Seed = 123
```

Comment:
Minor detail: If I understood correctly this value won't change, right? If that's the case we should probably change it to `val SeedNonce: Seed = 123`

Reply:
Resolved.